routing/payment_session_test.go

```diff
@@ -8,6 +8,54 @@ import (
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
+func TestValidateCLTVLimit(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		cltvLimit      uint32
```

Comment:
Could add a test name here to give a rough idea of what case is under test.

Reply:
added